base/applications/rapps/lang/tr-TR.rc

```diff
@@ -27,7 +28,7 @@ BEGIN
         MENUITEM SEPARATOR
         MENUITEM "&Yenile\tF5", ID_REFRESH
         MENUITEM SEPARATOR
-        MENUITEM "Uygulama Veri Ta&banını Yenile\tCtrl+F5", ID_RESETDB
+        MENUITEM "Uygulama Veri Tabanını Yenile\tCtrl+F5", ID_RESETDB
```

Comment:
No, don't strip that accelerator!

Reply:
look at my new commits I fixed the accelerator stages

> No, don't strip that accelerator!

Reply:
No you still didn't fix that.

Reply:
oh sorry my bad i changed now and changed deneme/tanıtım sürümü line in latest commit